text2pcap.c

```diff
@@ -50,57 +50,60 @@ const char *janus_text2pcap_packet_string(janus_text2pcap_packet type) {
 }
 
 janus_text2pcap *janus_text2pcap_create(const char *dir, const char *filename, int truncate) {
+	janus_text2pcap *tp;
+	char newname[1024];
+	char *fname;
+	FILE *f;
```

Comment:
All of the above pointers should be initialized to NULL.

Reply:
Actually, there is no need for it. If you follow the code, you'll see:

- `*tp` will be initialized with the allocation right before being used for the first time;
- `*f` will be initialized in both branches of the first `if`, also before the first use; and
- `*fname` will be initialized whenever `*f` is not set to `NULL` in the previous assignments, and when `*f` is in fact `NULL`, the code that uses `*fname` will never be reached.

But I admit the code is a bit confusing, so a refactoring is on the way. :)